dapps/marketplace/src/pages/create-listing/listing-types/FractionalHourlyListing/FractionalHourlyListing.js

```diff
@@ -5,13 +5,19 @@ import Details from './Details'
 import Boost from './Boost'
 import Availability from './Availability'
 import Review from './Review'
+import withWallet from '../../../../hoc/withWallet'
```

Comment:
the `hoc` dir has an alias in webpack, so you can import it with just `import withWallet from 'hoc/withWallet'`

Reply:
Didn't know about the alias. Have updated it now.